docs/utilities.md

```diff
@@ -1400,4 +1400,43 @@ One final important variable is ```.grafana.del```, this dictates the delimeter
 between options in the drop down menus. This has significant repercussions if 
 one of your columns includes full stops, eg. email adresses. As a result we have 
 left this as definable so that the user can alter this to a non-disruptive value 
-for their data eg./.
\ No newline at end of file
+for their data eg./.
+
+## cloudutils.q
+
+The cloud utilities allow other processes to be programmatically launched, or 
+terminated, from inside a TorQ process. The two main functions here are 
+.sys.launch and .sys.kill. 
+
+.sys.launch is the q function which takes a dictionary of input parameters which
+are then passed to a bash script. This bash script functions like torq.sh in how
+it starts processes. It is important to note that the cloud utilities are only
+supported on Linux as a result. 
+
+    `q)input:`procname`proctype`localtime`p!("hdb1";"hdb";"0";"1234");
+
+The input parameter dictionary, as shown above, should contain symbols as keys 
+and strings as values. Any standard or custom TorQ process can be launched 
+using .sys.launch, and as such the function can accept any desired command line
+parameters in the input dictionary. The minimum required are `procname and 
+`proctype. In the case that only these two are used the other arguments will be 
```

Comment:
```suggestion
parameters in the input dictionary. The minimum required are `` `procname`` and 
`` `proctype``. In the case that only these two are used the other arguments will be 
```

think you need to do this to prevent inline code formatting messing up here